src/formatDate.js

```diff
@@ -50,7 +50,34 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat[fromFormat.length - 1];
+  const newSeparator = toFormat[toFormat.length - 1];
+
+  const dateArray = date.split(oldSeparator);
+  const resultArray = [];
+  let resultString = '';
+
+  for (let i = 0; i < toFormat.length - 1; i++) {
```

Comment:
please put some spaces for you code, because code looks like unreadable right now